docs/src/operate/c8y/c8y_service_monitoring.md

```diff
@@ -11,59 +11,60 @@ or on the `child` device can be monitored from the **Cumulocity IoT** by sending
-
-|Device-type|Thin-edge-Health-status-mqtt-topic|Health-status-message|
-|------|------------------------|---------------------|
-|Thin-edge-device|`tedge/health/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
-|Child-device|`tedge/health/<child-device-id>/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
+A health status message can be published for any service on a `status/health` channel. The health message has to contain
+at least the `status` of the service.
 
 :::note
 The `status` here can be `up or down` or any other string. For example, `unknown`.
 :::
 
-For example, to monitor the health status of a `tedge-mapper-c8y` service that is running on a `thin-edge.io` device
-one has to send the below message.
-
-```sh te2mqtt
-tedge mqtt pub tedge/health/tedge-mapper-c8y '{"status":"up","type":"thin-edge.io"}' -q 2 -r
-```
-
-The above message says that the `tedge-mapper-c8y` is `up` and the `type` of the service is `thin-edge.io`.
+For example, to update the health status of `device/my-device/service/my-test-service` service, one has to send the
+following message:
 
-
-To monitor the health of a `docker` service that is running on an `external-sensor` child device,
-
-```sh te2mqtt
-tedge mqtt pub tedge/health/external-sensor/docker '{"status":"up","type":"systemd"}' -q 2 -r
+```sh te2mqtt formats=v1
+tedge mqtt pub te/device/my-device/service/my-test-service/status/health '{"status":"up"}' -q 2 -r
 ```
 
 :::note
 The health status message has to be sent as a `retain` message.
 :::
 
-When an empty health status message is sent, e.g. `{}` or `''`, the `status` will be replaced with `unknown` and the `type` will be replaced with default value `service`.
+When an empty health status message is sent, e.g. `{}` or `''`, the `status` will be replaced with `unknown`.
 
 ## Conversion of the health status message to Cumulocity IoT service monitor message
 
-The `tedge-mapper-c8y` will translate the `health status` message that is received on `tedge/health/#`
-topic to `Cumulocity` specific `service monitor` message and sends it to `Cumulocity` cloud.
+The `tedge-mapper-c8y` will translate any health status message that is received on `te/+/+/+/+/status/health` topic to
+Cumulocity [Service status update](https://cumulocity.com/guides/reference/smartrest-two/#104) SmartREST message and
+send it to the `Cumulocity` cloud. If a service was not previously registered and it fulfills the requirements for
+auto-registration, it will be auto-registered as described [in the Auto Registration
+section](https://thin-edge.github.io/thin-edge.io/next/references/mqtt-api/#auto-registration).
 
-The table below gives more information about the **Cumulocity IoT** topic and the translated service monitor message for both `thin-edge` as well as for `child` device.
+For example, assuming a service `device/child1/service/service1`, running on a device `device/child1//`, which is a
```

Comment:
Very small thing to keep consistency. In the following example input and output, you use `child` instead of `child1`. So, either chaning this line from `child1` to `child`, or the example from `child` to `child1`. 

```suggestion
For example, assuming a service `device/child/service/service1`, running on a device `device/child//`, which is a
```